app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
```

Comment:
why are we using this home controller? 
is this redirecting to the home page or something like that after logging in?

Reply:
Yes bhaia, this controller is used for Home Page.

Reply:
Why we are not using the user controller? 
Do we actually need a separate controller to just redirect to a single action when the home page is related to the user's home page and we already have a controller for users?

Reply:
Thanks for the feedback vaia, removed the HomeController and configured root path to Users Index Page.